bm-instance-agent/bm_instance_agent/objects.py

```
@@ -178,7 +178,8 @@ class BondPortParasObj(Base):
 
     @classmethod
     def from_json(cls, paras):
-
+        # TODO: -> python3:
+        # elif isinstance(paras, str):
         if isinstance(paras, (str, unicode)):
```

Ruff (0.15.4)

[error] Undefined name `unicode`

(F821)